newIDE/app/public/external/yarn/.gitignore

```diff
@@ -0,0 +1,87 @@
+.idea/
```

Comment:
This .gitignore is good but a bit too complete, let's instead put only the things useful for yarn (other things like .idea/grun/OSX/thumbnails should in theory be handled, if necessary, by the root .gitignore of the GD repo)

Reply:
no probs, just following the convention established with the other ones :)

Reply:
this file is actually not needed. It was autocreated by vscode I think. The gitignore is handled elsewhere, so I will delete this